spec/index.html

```diff
@@ -371,26 +372,41 @@ <h3>Triple Terms and Reification</h3>
       </a>
       <figcaption id="fig-asserted-triple-term-alt">
         An <a>RDF graph</a> containing a <a>triple annotation</a>, where the <a>triple term</a> of a <a>reifying triple</a> corresponds to an <a>asserted triple</a>.
-        The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
+        Due to the asserted triple, the diagram represents the proposition as a fact, meaning that the relationship holds.
       </figcaption>
     </figure>
 
+    <p>
+      Concrete syntaxes, such as Turtle [[RDF12-TURTLE]], may have shortcuts
+      for specifying <a>reifying triples</a> and <a>triple annotations</a> more
+      succinctly.
+    </p>
+ 
+    <p>Finally, [=RDF terms=] that [=appear=] in a [=triple term=] have the same
+      <a data-lt="denotes">denotation</a> as when they appear in an
+      [=asserted triple=] in the <a data-lt="RDF graph">graph</a>.
+For example, the term <code>:Alice</code> from a [=triple term=] and <code>:Alice</code> in an [=asserted triple=] both denote the same resource.
+      For this reason, we say that triple terms are
+      <dfn class="lint-ignore">transparent</dfn>.</p>
+
     <div class="note">
       <p><a>Triple terms</a> always denote abstract, logical <a>propositions</a>,
         while <a>reifiers</a> may denote a variety of things that are related
         to these propositions (such as a statement or belief that the
         proposition holds, or an event or circumstance that makes the
-        proposition true).
-        It is therefore expected that triple terms are commonly used as
+        proposition true). 
+        It is expected that triple terms are commonly used as
         <a>objects</a> of <a>reifying triples</a>, and that the reifiers
-        (rather than the triple terms) are used in further descriptions.</p>
+        (rather than the triple terms) are used in further descriptions.
+        <a href="#section-triple-terms-reification">This section</a> describes this common usage.
+      </p>
       <p>
         There can be multiple, distinct reifiers related to the same abstract
         proposition, such as statements with different sources, or
         circumstances with different characteristics. One reifier may also be
-        used to reify multiple, distinct propositions, expressing different
-        propositional aspects using the same reifier.</p>
-      <p>Since a proposition so reified does not have to hold, it is possible
+        used to reify multiple, distinct propositions, 
+        expressing different propositional aspects using the same reifier.</p>
```

Comment:
```suggestion
        used to reify multiple, distinct propositions, 
        expressing for example the fact that the same situation could stem from different propositions.</p>
```
@franconi's 2nd suggestion above, with two minor changes
* added "for examples" (it could just as well be a belief that several propositions hold at the same time)
* replaced "come from" with "stem from"